Classes/vendor/MYUtilities/MYErrorUtils.m

```diff
@@ -139,22 +139,19 @@ BOOL MYMiscError( NSError **error, NSString *message, ... )
 #if !TARGET_OS_IPHONE || defined(__SEC_TYPES__)
     else if ($equal(domain,NSOSStatusErrorDomain)) {
         // If it's an OSStatus, check whether CarbonCore knows its name:
-        const char *name = NULL;
-#if !TARGET_OS_IPHONE
-        name = GetMacOSStatusErrorString((int)code);
-#endif
-        if (name && *name)
-            result = [NSString stringWithCString: name encoding: NSMacOSRomanStringEncoding];
-        else {
+        NSError *osErr = [NSError errorWithDomain:NSOSStatusErrorDomain code:code userInfo:nil];
+        result = [osErr localizedDescription];
```

Comment:
If we return the localised description, what would happen if we run the test with another language (not english)? I mean, will [these tests](https://github.com/cloudant/CDTDatastore/blob/master/Classes/vendor/MYUtilities/MYErrorUtils.m#L212) pass?